app/assets/stylesheets/admin/loans/_questionnaires.scss

```diff
@@ -71,6 +71,10 @@ section.questionnaires {
     margin-bottom: 1em;
   }
 
+  .note-editable li, .answer li {
+    min-height: 1em;
```

Comment:
you may also want to set the margin-bottom to `0` here, i don't think list items should have quite that much space between them?